src/Symfony/Component/Testing/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "symfony/testing",
+    "type": "library",
+    "description": "Symfony Testing Component",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "http://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.3"
```

Comment:
sorry, I overlooked the original comment on the "outdated diff", where it is sated that it is intentionally, so my last comment can be ignored